src/lammpsio/topology.py

```diff
@@ -258,13 +357,37 @@ def __init__(self, N, num_types=None):
 
 
 class LabelMap(collections.abc.MutableMapping):
-    """Label map between typeids and types.
+    """Map between integer type IDs and string type names.
+
+    A `LabelMap` is effectively a dictionary associating a label (type) with a
+    particle's or connection's typeid. These labels can be useful for tracking
+    the meaning of typeids. They are also automatically used when interconverting
+    with HOOMD GSD files that require such labels.
 
     Parameters
     ----------
     map : dict
         Map of typeids to types.
 
+    Example
+    -------
+    Create `LabelMap`:
+
+    .. code-block:: python
+
+        snapshot = lammpsio.Snapshot(
+            N=3,
+            box=lammpsio.Box([-5, -5, -5], [5, 5, 5]),
+            step=10
+        )
+        snapshot.type_label = lammpsio.topology.LabelMap({1: "A", 2: "B"})
+        snapshot.typeid = [1, 2, 1]
+        snapshot.types = ["A", "B", "A"]
+
+    This creates a dictionary mapping numeric type ID labels 1 and 2 used by LAMMPS
+    to alphanumeric type ID labels "A" and "B" used by HOOMD-blue to define the
+    different particle types.
```

Comment:
This example seems off to me. It's focused more on the snapshot on the label map? The `Snapshot` also doesn't have a `types` attribute, does it? I think it can just be the example you showed:
```suggestion
    .. code-block:: python

        type_label = lammpsio.topology.LabelMap({1: "A", 2: "B"})

    This creates a dictionary mapping numeric type ID labels 1 and 2 used by LAMMPS
    to alphanumeric type labels "A" and "B", such as those used by HOOMD-blue.
```